src/Sentry.Tunnel/SentryTunnelMiddleware.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Net.Http.Headers;
+
+namespace Sentry.Tunnel
+{
+    public class SentryTunnelMiddleware : IMiddleware
```

Comment:
all release builds require xml docs on all public members (otherwise we keep forgetting to document the API). Could you please add some like:

```suggestion
    // <summary>
    // Middleware that can forward Sentry envelopes.
    // </summary>
    // <seealso href="https://docs.sentry.io/platforms/javascript/troubleshooting/#dealing-with-ad-blockers">
    public class SentryTunnelMiddleware : IMiddleware
```